core/common/src/geometry/ElementGeometryFB.ts

```diff
@@ -12,7 +12,6 @@ import { flatbuffers } from "flatbuffers";
 /* eslint-disable no-var */
 // cspell:word flatbuffers
 
-/** @internal */

```

Comment:
@bbastings is there any reason we'd want the generated flatbuffer goop included in the published, `@public` API documentation?

Reply:
Probably not, and it is not included in the barrel file. I fixed that with #2032. However, there is an existing docs bug that was unearthed with that change. [Notice how localToWorld links to EGFBAccessors.Transform instead of the Transform in the geometry package](https://www.itwinjs.org/reference/imodeljs-common/geometry/geometrystreamiteratorentry/). Some bug I'm still working through, but this PR puts things back how they were and will unblock docs builds in the meantime.

Reply:
Probably not wanted, people will be happier using the ElementGeometry api.

Reply:
Can you file a work item somewhere to put this back to `@internal` once you've fixed the docs bug?

Reply:
I was able to fix this by marking Transform as internal. I assume that is correct as it is part of the `EGFBAccessors` namespace?